webapp/components/post_view/post_attachment_opengraph/post_attachment_opengraph.jsx

```diff
@@ -208,8 +222,39 @@ export default class PostAttachmentOpenGraph extends React.PureComponent {
         return text;
     }
 
+    handleRemovePreview() {
+        const props = Object.assign({}, this.props.post.props);
```

Comment:
the way I see it, `post` is not a required prop and so it can be undefined, so when accessing the `props` of an undefined object can cause JS errors